packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```diff
@@ -173,9 +173,659 @@
+          "publicKey": {
+            "type": "string"
+          }
+        }
+      },
+      "RevokePermissionRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Revoke Permission.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "permission"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "permission": {
+            "type": "number"
+          }
+        }
+      },
+      "SetAccountDetailRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Set Account Detail.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "key",
+          "value"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "key": {
+            "type": "string"
+          },
+          "value": {
+            "type": "string"
+          }
+        }
+      },
+      "SetAccountQuorumRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Set Account Quorum.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "quorum"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "quorum": {
+            "type": "number"
+          }
+        }
+      },
+      "SubtractAssetQuantityRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Subtract Asset Quantity.",
+        "type": "object",
+        "required": [
+          "assetId",
+          "amount"
+        ],
+        "properties": {
+          "assetId": {
+            "type": "string"
+          },
+          "amount": {
+            "type": "number"
+          }
+        }
+      },
+      "TransferAssetRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Transfer Asset.",
+        "type": "object",
+        "required": [
+          "srcAccountId",
+          "destAccountId",
+          "assetId",
+          "description",
+          "amount"
+        ],
+        "properties": {
+          "srcAccountId": {
+            "type": "string"
+          },
+          "destAccountId": {
+            "type": "string"
+          },
+          "assetId": {
+            "type": "string"
+          },
+          "description": {
+            "type": "string"
+          },
+          "amount": {
+            "type": "number"
+          }
+        }
+      },
+      "CompareAndSetAccountDetailRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Compare And Set Account Detail.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "key",
+          "value"
+        ],
```

Comment:
According to: https://iroha.readthedocs.io/en/develop/develop/api/commands.html#compare-and-set-account-detail also `check_empty` is required